src/SamplesApp/UITests.Shared/Windows_Devices/PedometerTests.xaml.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Windows.Input;
+using Uno.Disposables;
+using Uno.UI.Samples.Controls;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.Devices.Sensors;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_Devices
+{
+	[SampleControlInfo(
+		"Windows.Devices",
+		"Pedometer",
+		description: "Demonstrates the Windows.Devices.Sensors.Pedometer",
+		viewModelType: typeof(PedometerTestsViewModel))]
+    public sealed partial class PedometerTests : UserControl
+    {
+        public PedometerTests()
+        {
+            this.InitializeComponent();
+        }
+    }
+
+	public class PedometerTestsViewModel : ViewModelBase
+	{
+		private Pedometer _pedometer = null;
+		private string _pedometerStatus;
+		private bool _readingChangedAttached;
+		private int _cumulativeSteps;
+		private double _cumulativeStepsDurationInSeconds;
+		private string _timestamp;
+
+		public PedometerTestsViewModel(CoreDispatcher dispatcher) :
+			base(dispatcher)
+		{
+		}
+
+		public ICommand GetPedometerCommand => GetOrCreateCommand(GetPedometerAsync);
+
+		public string PedometerStatus
+		{
+			get => _pedometerStatus;
+			private set
+			{
+				_pedometerStatus = value;
+				RaisePropertyChanged();
+			}
+		}
+
+		public bool IsAvailable => _pedometer != null;
+
+		private async void GetPedometerAsync()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Return 'Task' instead.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826529)

Reply:
Which is not supported by the `GetOrCreateCommand`. We could add it, but we would face the question of how to handle multiple parallel executions of the command, how to handle errors, etc. ... for tests.

Reply:
I would think this could be ok for the sample.